pkg/licensing/normalize.go

```diff
@@ -181,7 +169,14 @@ var licenseSplitRegexp = regexp.MustCompile("(,?[_ ]+(?:or|and)[_ ]+)|(,[ ]*)")
 
 func Normalize(name string) string {
 	name = strings.TrimSpace(name)
```

Comment:
let's add a new variable for the updated license name to avoid renaming the license if we don't find that license in the `mapping`
```suggestion
	license := strings.TrimSpace(name)
```

See https://github.com/aquasecurity/trivy/actions/runs/9888626513/job/27317191927?pr=7131#step:9:2387

Reply:
I also performed additional space and spelling normalizations and added many new mappings from [oss-review-toolkit](https://github.com/oss-review-toolkit/ort/tree/fc5389c2cfd9c8b009794c8a11f5c91321b7a730/utils/spdx/src/main/resources)

Reply:
> I need to fix integration tests.

You can use `mage test:updateGolden` command.